tests/CitizenProject/CitizenProjectMessageNotifierTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Tests\AppBundle\CitizenProject;
+
+use AppBundle\CitizenProject\CitizenProjectMessageNotifier;
+use AppBundle\CitizenProject\CitizenProjectWasApprovedEvent;
+use AppBundle\CitizenProject\CitizenProjectWasCreatedEvent;
+use AppBundle\DataFixtures\ORM\LoadCitizenProjectData;
+use AppBundle\CitizenProject\CitizenProjectManager;
+use AppBundle\DataFixtures\ORM\LoadAdherentData;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CitizenProject;
+use AppBundle\Mailer\MailerService;
+use OldSound\RabbitMqBundle\RabbitMq\ProducerInterface;
+use PHPUnit\Framework\TestCase;
+use Ramsey\Uuid\Uuid;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+
+/**
+ * @functional
```

Comment:
`@group functional` ?
not needed though as these tests are unit tests

Reply:
✔️